src/egon/data/datasets/industrial_gas_demand.py

```diff
@@ -453,14 +470,31 @@ def import_industrial_gas_demand_time_series(egon_etrago_load_gas):
     )
 
 
-def insert_industrial_gas_demand():
-    """Overall function for inserting the industrial gas demand
+def download_industrial_gas_demand():
+    """Download the industrial gas demand data from opendata.ffe database."""
+    correspondance_url = (
+        "http://opendata.ffe.de:3000/region?id_region_type=eq.38"
+    )
 
-    Returns
-    -------
-    None.
-    """
+    # Read and save data
+    result_corr = requests.get(correspondance_url)
+    pd.read_json(result_corr.content).to_json("region_corr_ind_demand.json")
+
+    carriers = {"H2": "2,162", "CH4": "2,11"}
+    url = "http://opendata.ffe.de:3000/opendata?id_opendata=eq.66&&year=eq."
 
-    egon_etrago_load_gas = import_industrial_gas_demand(scn_name="eGon2035")
+    for scn_name in ["eGon2035", "eGon100RE"]:
+        year = str(
+            get_sector_parameters("global", scn_name)["population_year"]
+        )
+
+        for carrier, internal_id in carriers.items():
+            # Download the data
+            datafilter = "&&internal_id=eq.{" + internal_id + "}"
+            request = url + year + datafilter
 
-    import_industrial_gas_demand_time_series(egon_etrago_load_gas)
+            # Read and save data
+            result = requests.get(request)
+            pd.read_json(result.content).to_json(
+                carrier + "_ind_demand_" + scn_name + ".json"
+            )

```

Comment:
These files could be stored in datasets/gas_data

Reply:
Fixed by https://github.com/openego/eGon-data/pull/639/commits/71e94fad9dcaf7124ffdaf8054b313f743fe984d